test/org/sagebionetworks/bridge/services/AuthenticationServiceImplTest.java

```diff
@@ -57,6 +76,11 @@
     @Before
     public void before() {
         testUser = helper.createUser(AuthenticationServiceImplTest.class);
+        Study study = studyService.getStudy(TestConstants.TEST_STUDY_IDENTIFIER);
+        if (study.getDataGroups().isEmpty()) {
+            study.getDataGroups().add("foo");
+            studyService.updateStudy(study, true);
+        }
```

Comment:
It might be cleaner to say !study.getDataGroups().contains("foo"). This will make other tests simpler.